ui/app/css/itcss/components/swap.scss

```diff
@@ -607,7 +607,6 @@
 
     span {
       color: #2f9ae0;
-      cursor: pointer;
```

Comment:
"Security & Privacy" is not a link anymore, so I removed the pointer. The colour still remains so it looks good. Let me know if I should remove the colour as well.

<img width="413" alt="Screenshot 2021-07-07 at 19 21 31" src="https://user-images.githubusercontent.com/3684187/124771223-40504900-df3b-11eb-99e7-3c7240454aba.png">